pkg/eniconfig/eniconfig_test.go

```diff
@@ -144,3 +171,44 @@ func TestNodeENIConfig(t *testing.T) {
 	assert.Equal(t, defaultCfg, *outputCfg)
 
 }
+
+func TestNodeENIConfigLabel(t *testing.T) {
```

Comment:
Same with above comment. Is it not possible to merge this test into `TestNodeENIConfig()`?

Reply:
I'm planning a bigger refactor of the test code so I am OK with this.